apps/avatax/src/modules/avatax/calculate-taxes/avatax-calculate-taxes-response-shipping-transformer.ts

```diff
@@ -1,61 +1,67 @@
 import { TransactionModel } from "avatax/lib/models/TransactionModel";
 import { numbers } from "../../taxes/numbers";
+import { TaxBadProviderResponseError } from "../../taxes/tax-error";
 import { taxProviderUtils } from "../../taxes/tax-provider-utils";
 import { CalculateTaxesResponse } from "../../taxes/tax-provider-webhook";
-import { SHIPPING_ITEM_CODE } from "./avatax-calculate-taxes-adapter";
-import { TaxBadProviderResponseError } from "../../taxes/tax-error";
-
-export class AvataxCalculateTaxesResponseShippingTransformer {
-  transform(
-    transaction: TransactionModel,
-  ): Pick<
-    CalculateTaxesResponse,
-    "shipping_price_gross_amount" | "shipping_price_net_amount" | "shipping_tax_rate"
-  > {
-    const shippingLine = transaction.lines?.find((line) => line.itemCode === SHIPPING_ITEM_CODE);
+import { createLogger } from "@saleor/apps-logger";
+import { avataxShippingLine } from "./avatax-shipping-line";
 
-    if (!shippingLine) {
-      return {
-        shipping_price_gross_amount: 0,
-        shipping_price_net_amount: 0,
-        shipping_tax_rate: 0,
-      };
-    }
+const logger = createLogger("transformAvataxTransactionModelIntoShipping");
 
-    if (!shippingLine.isItemTaxable) {
-      return {
-        shipping_price_gross_amount: taxProviderUtils.resolveOptionalOrThrowUnexpectedError(
-          shippingLine.lineAmount,
-          new TaxBadProviderResponseError("shippingLine.lineAmount is undefined"),
-        ),
-        shipping_price_net_amount: taxProviderUtils.resolveOptionalOrThrowUnexpectedError(
-          shippingLine.lineAmount,
-          new TaxBadProviderResponseError("shippingLine.lineAmount is undefined"),
-        ),
-        /*
-         * avatax doesn't return combined tax rate
-         * // todo: calculate percentage tax rate
-         */
-        shipping_tax_rate: 0,
-      };
-    }
+// why is tax rate 0?
```

Comment:
what should we do with this comment?

Reply:
I created a ticket for it https://linear.app/saleor/issue/SHOPX-237/map-the-shipping-tax-rate-field-in-x-calculate-taxes-response-in